pyqt-apps/siriushla/li_rf_llrf/widgets.py

```diff
@@ -55,3 +61,216 @@ def _handle_enable_state(self):
             self.ch_loop_enable.connected and  # is connected
             self.ch_loop_enable.value == 0)    # is disabled
         self.setEnabled(state)
+
+
+class GraphIvsQ(QWidget):
+    """Show the I&Q Chart based on IxQ or Time"""
+
+    def __init__(
+        self, parent=None, dev=None,
+        prop='Time', main_dev='', channel='CH1', prefix=''):
+            """."""
+            super().__init__(parent=parent)
+            self.prefix = prefix
+            self.main_dev = main_dev
+            self.dev = dev
+            self.prop = prop
+            self.channel = channel
+            self._setupui()
+
+    def _setupui(self):
+        """."""
+        channels = {}
+        lay1 = QGridLayout()
+        self.setLayout(lay1)
+
+        graph = SiriusWaveformPlot(self)
+        graph.setObjectName('graph')
+        graph.setStyleSheet('#graph {min-height: 15em; min-width: 20em;}')
+        graph.maxRedrawRate = 2
+        graph.mouseEnabledX = True
+        graph.setShowXGrid(True)
+        graph.setShowYGrid(True)
+        graph.setBackgroundColor(QColor(_util.get_appropriate_color('LI')))
+        graph.setShowLegend(True)
+        graph.setAutoRangeX(False)
+        graph.setAutoRangeY(False)
+        graph.plotItem.showButtons()
```

Comment:
this is no longer needed here since `SiriusWaveformPlot` already does it

Reply:
This is necessary if you want to set the maximum and minimum ranges to 1 and -1 on the axis of the fasor chart.